src/spacesettlers/simulator/Toroidal2DPhysics.java

```diff
@@ -1037,9 +1037,9 @@ public void toroidalWrap(Position position) {
 
 	/**
 	 * Respawns any dead objects in new random locations. Ships have a delay before
-	 * they can respawn.
+	 * they can respawn.  Asteroids do not respawn (they are re-created later randomly)
 	 */
-	public void respawnDeadObjects(Random random, double asteroidMaxVelocity) {
+	public void respawnDeadObjects(Random random) {
```

Comment:
Maybe have it create a new mineable asteroid and give it a new location in here when an asteroid should respawn?

Reply:
Yeah, I think there is a cleaner way.  Looking through code that is old and that's how I accidentally cleaned it up and got rid of the probabilities.  Trying to find the cleanest way.  May be to go back to re-spawning asteroids (as I was doing) to ensure their properties stay the same but just randomizing their radius and contents.